addons/medical_vitals/functions/fnc_handleUnitVitals.sqf

```diff
@@ -126,7 +126,7 @@ switch (true) do {
             [QEGVAR(medical,CriticalVitals), _unit] call CBA_fnc_localEvent;
         };
     };
-    case (_bloodLoss > BLOOD_LOSS_KNOCK_OUT_THRESHOLD * _cardiacOutput): {
+    case (_bloodLoss / EGVAR(medical,bleedingCoefficient) > BLOOD_LOSS_KNOCK_OUT_THRESHOLD * _cardiacOutput ): {
```

Comment:
extra space before bracket. 

All in all nice changes!

Reply:
Thanks! Space should be fixed now.